v1/lib/static/css/main.css

```diff
@@ -2220,14 +2220,14 @@ input::placeholder {
 }
 
 .showcaseSection .logos img {
-  max-height: 128px;
+  height: 128px;
```

Comment:
This will break logos that aren't square in proportion. For example, have a look at Vuls and ml5.js on https://deploy-preview-1043--docusaurus-preview.netlify.com/en/users and compared them with https://docusaurus.io/en/users. The images are distorted.

You could add a wrapper div that has fixed width and height around all the images so that you can align the names and still maintain the aspect ratio of the images.

Reply:
@yangshun Updated, waited for the deploy env to be built - but it is on hold for some reason